src/EdFi.Admin.LearningStandards.CLI/Internal/LearningStandardsCLIApplication.cs

```diff
@@ -43,11 +41,11 @@ internal LearningStandardsCLIApplication(Action<IServiceCollection> serviceSetup
             _serviceSetup = serviceSetup;
         }
 
-        internal LearningStandardsCLIWriter CliWriter { get; private set; }
+        public bool Unattended => _options.Unattended;
 
-        internal bool Unattended => _options.Unattended;
+        public LearningStandardsCLIWriter CliWriter { get; set; }
```

Comment:
Are these used outside the class, or could they be private/internal?

Reply:
yes, those properties are used in program.cs file